sap-2-tier-marketplace-image/shared/os-disk-parts.json

```diff
@@ -100,7 +101,7 @@
             "uri": "[parameters('userImageVhdUri')]"
           },
           "vhd": {
-            "uri": "[concat('http://',parameters('userImageStorageAccount'),'.blob.core.windows.net/', resourceGroup().name, '/', parameters('vmName'), '-', variables('osDiskName'))]"
+            "uri": "[concat('http://',parameters('userImageStorageAccount'),'.blob.core.windows.net/', concat('vhds-', uniqueString(resourceGroup().id)), '/', parameters('vmName'), '-', variables('osDiskName'))]"
```

Comment:
I will make a bigger reorg of all sap templates soon. Can you in the meantime merge this PR?

Thanks